plotly/tests/test_core/test_tools/test_Quiver.py

```diff
@@ -0,0 +1,223 @@
+import plotly
+from plotly.graph_objs import *
+import plotly.tools as tls
+from nose.tools import raises
+import numpy as np
+
+
+@raises(Exception)
+def unequal_xy_length():
+    data = tls.Quiver(x=[1, 2], y=[1], u=[1, 2], v=[1, 2])
+
+
+@raises(Exception)
+def unequal_uv_length():
+    data = tls.Quiver(x=[1, 2], y=[1, 3], u=[1], v=[1, 2])
+
+
+@raises(Exception)
+def test_wrong_kwarg():
```

Comment:
Also, it's usually more powerful to test via `TestCase`, which you get from `from unittest import TestCase`.

Here's an example in this repo:
https://github.com/plotly/python-api/blob/4a19ce2d382495692affd47dd107baab1755f5a7/plotly/tests/test_core/test_plotly/test_credentials.py#L33-73